packages/Constants/README.md

```diff
@@ -0,0 +1,4 @@
+<!-- autogenerated don't modify -->
```

Comment:
add explanation and disclaimer about the package is for internal use exclusively and to access the constants of a component they can do it over `Component.types`